docs/openssl.md

```diff
@@ -36,15 +36,17 @@ OpenSSL support is enabled by default for all supported algorithms. If you want
 Each algorithm can be disabled individually by setting the following system properties on the command line:
 
 
-- To turn off **Digest**, set `-Djdk.nativeDigest=false`
-- To turn off **ChaCha20** and **ChaCha20-Poly1305**, set `-Djdk.nativeChaCha20=false`. :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** ![Start of content that applies to Java 8 (LTS)](cr/java8.png) These algorithms are not supported on Java 8 ![End of content that applies only to Java 8 (LTS)](cr/java_close_lts.png)
-- To turn off **CBC**, set `-Djdk.nativeCBC=false`
-- To turn off **GCM**, set `-Djdk.nativeGCM=false`
-- To turn off **RSA**, set `-Djdk.nativeRSA=false`
-- To turn off **ECDH key agreement**, set `-Djdk.nativeEC=false`
-- To turn off **EC key generation**, set `-Djdk.nativeECKeyGen=false`
-- ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) To turn off **XDH key agreement**, set `-Djdk.nativeXDHKeyAgreement=false`
-- To turn off **XDH key generation**, set `-Djdk.nativeXDHKeyGen=false` ![End of content that applies to Java 11 (LTS) and later](cr/java_close_lts.png)
+- To turn off **Digest**, set [`-Djdk.nativeDigest=false`](djdknativedigest.md)
+- To turn off **ChaCha20** and **ChaCha20-Poly1305**, set [`-Djdk.nativeChaCha20=false`](djdknativechacha20.md). :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** ![Start of content that applies to Java 8 (LTS)](cr/java8.png) These algorithms are not supported on Java 8 ![End of content that applies only to Java 8 (LTS)](cr/java_close_lts.png)
+- To turn off **CBC**, set [`-Djdk.nativeCBC=false`](djdknativecbc.md)
+- To turn off **ECDH key agreement**, set [`-Djdk.nativeEC=false`](djdknativeec.md)
+- To turn of **ECDSA signature**, set [`-Djdk.nativeECDSA=false`](djdknativeecdsa.md)
+- To turn off **EC key generation**, set [`-Djdk.nativeECKeyGen=false`](djdknativeeckeygen.md)
```

Comment:
EC should come before ECDH

Reply:
@pshipton - As per the IBM style guide, "Use the letter-by-letter method, which ignores spaces." while sorting. So, if **EC key generation** is the name of the algorithm then as per the guidelines, **EC key generation** should come after **ECDSA signature**. 

Do you still prefer to change the order as you have suggested? Any specific reason? Thanks!

Reply:
I would have sorted EC before ECDSA, but pls match the IBM style guide.